db/migrate/20230328220723_create_post.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
as required you should have a migration file for creating a posts table as seen below 
![image](https://user-images.githubusercontent.com/60612329/228525717-bfc73e22-64c2-4431-95d0-9d7b55118f1b.png)

Reply:
hi @Wahaj-Ali i apologise for not cross checking. submit it and it will be approved

Reply:
@alexander16108 thank you for your cooperation 😄